build.gradle

```diff
@@ -0,0 +1,89 @@
+subprojects {
+    apply plugin: 'java'
+    apply plugin: 'scala'
+
+    apply from: "${rootProject.projectDir}/gradle-scalatest.plugin"
+
+    ext {
+        ////////////////////////////////
+        // Commonly used dependencies //
+        ////////////////////////////////
+
+        mkDepGen = { genOne -> { String... names -> names.collect(genOne) } }
```

Comment:
Alright, I'll switch over to inline dependency declarations. I think introducing version rewriting like you suggest will cause confusion, I'd rather keep the version numbers in `gradle.properties`. Would that work for you?